content/en/synthetics/cicd_integrations/_index.md

```diff
@@ -1,400 +1,58 @@
-
-`deviceIds`
-: **Type**: array<br>
-List of devices on which to run the browser test.
-
-`followRedirects`
-: **Type**: boolean<br>
-Indicates whether to follow redirections in HTTP tests.
-
-`headers`
-: **Type**: object<br>
-Headers to replace in the HTTP or browser test. This object should contain, as keys, the name of the header to replace and, as values, the new value of the header.
-
-`locations`
-: **Type**: array<br>
-List of locations from which the test should be run.
-
-`retry`
-: **Type**: object<br>
-Retry policy for the test:
-  - `count`: integer. Number of attempts to perform in case of test failure.
-  - `interval`: integer. Interval between the attempts (in milliseconds).
-
-`executionRule`
-: **Type**: string<br>
-Execution rule of the test: defines the behavior of the CLI in case of a failing test:
-  - `blocking`: The CLI returns an error if the test fails.
-  - `non_blocking`: The CLI only prints a warning if the test fails.
-  - `skipped`: The test is not executed at all.
-
-`startUrl`
-: **Type**: string<br>
-New start URL to provide to the HTTP or browser test.
-
-`variables`
-: **Type**: object<br>
-Variables to replace in the test. This object should contain, as keys, the name of the variable to replace and, as values, the new value of the variable.
-
-`pollingTimeout`
-: **Type**: integer<br>
-The duration in milliseconds after which `datadog-ci` stops polling for test results. The default is 120,000 ms. At the CI level, test results completed after this duration are considered failed.
-
-**Note**: Tests' overrides take precedence over global overrides.
-
-**Example advanced test configuration file**:
-
-```json
-{
-    "tests": [
-        {
-            "id": "<TEST_PUBLIC_ID>",
-            "config": {
-                "allowInsecureCertificates": true,
-                "basicAuth": { "username": "test", "password": "test" },
-                "body": "{\"fakeContent\":true}",
-                "bodyType": "application/json",
-                "cookies": "name1=value1;name2=value2;",
-                "deviceIds": ["laptop_large"],
-                "followRedirects": true,
-                "headers": { "<NEW_HEADER>": "<NEW_VALUE>" },
-            "locations": ["aws:us-west-1"],
-                "retry": { "count": 2, "interval": 300 },
-                "executionRule": "skipped",
-                "startUrl": "{{URL}}?static_hash={{STATIC_HASH}}",
-                "variables": { "titleVariable": "new value" },
-                "pollingTimeout": 180000
-            }
-        }
-    ]
-}
-```
+{{< whatsnext desc="With Synthetics and CI/CD, you can run Synthetic tests in the CI platform provider of your choice:" >}}
+    {{< nextlink href="synthetics/cicd_integrations/jenkins" >}}Jenkins{{< /nextlink >}}
+    {{< nextlink href="synthetics/cicd_integrations/github_actions" >}}GitHub Actions{{< /nextlink >}}
+{{< /whatsnext >}}
 
-#### Execution rule
+## Use the API
```

Comment:
Can we put the paragraph on using the CLI (line 85) first and using the API underneath? LMK what you think 😄